### ISSUE TITLE
Why pinning Node and Yarn versions?

### ISSUE THREAD

**Issue by 2075** [hoprnet/hopr-chat#24]
**Describe the bug**

While understanding reasons for pinning versions in general, 
when building, package settings for hopr-chat and hopr-core and others force you to downgrade node and yarn to older versions (while e.g. contained packages are not pinned). Is there a breaking change requiring?

Otherwise I would recommend to either use LTS or loosen up the pinned versions to either ~ or >.

**To Reproduce**
pull repo
yarn

**Expected behavior**
build without complaint

**Comment by jjperezaguinaga** [comment]
@2075 When we were building the binaries, we were exploring [nexe](https://github.com/nexe/nexe/releases) and [pkg](https://github.com/vercel/pkg) to help us bundle everything we needed for a desktop application. Sadly in some cases, they only had pre-compiled versions for specific node versions which made testing quite hard, as all our dependencies had quite loose versioning for node. Thus, we pinned both versions to allow us to have a working standard to work on across our repos.

We ended up getting having to stick with node.js and use [ncc](https://github.com/vercel/ncc) for producing "binaries". We have yet to solve the problem of bundling node.js with our binaries to avoid having the user to download any requirements. In the meantime, we realize that for development this has been a bit of a hurdle and we'll be loosing these versions soon.

In the meantime, we suggest using `yarn policies set-version 1.19.2` or using [yvm](http://yvm.js.org/) and do `yvm use` to set yarn to the right version. We are updating all our deps so we should be able to avoid pinning anymore EOM.

**Comment by 2075** [comment]
yes, i thought so, too. did not see pinning of yarn often (never) yet, both, when you are not exclusively working on hopr code make testing hard, as usually people knowing bash-fu do other work as well. maybe easier to use a shell inside a docker for testing. saw your constraints with "windows", dont know many who are using that for such development, though :D but docker works as well over there.

looking forward to EOM then. ;)
